DOKS-CI-CD/README.md

```diff
@@ -0,0 +1,1340 @@
+  - `tekton-argocd-build-deploy.yaml` - contains the definition for the CI/CD pipeline (in-depth explanations can be found inside).
+- `tasks` - contains configuration files for custom Tekton Tasks used in this tutorial. Following manifests are present here:
+  - `argocd-task-create-app.yaml` - defines the Argo CD task for creating new applications.
+- `triggers` - contains configuration files for the Tekton Triggers used in this tutorial. Following manifests are present here:
+  - `rbac.yaml` - defines the service account and required role bindings used by the Tekton EventListener from this tutorial. This is required by the EventListener to instantiate resources, such as the Tekton CI/CD pipeline.
+  - `tekton-argocd-build-deploy-event-listener.yaml` - contains the definition for the Tekton EventListener used this tutorial (in-depth explanations can be found inside).
+  - `tekton-argocd-build-deploy-trigger-binding.yaml` - contains the definition for the Tekton TriggerBinding used this tutorial (in-depth explanations can be found inside).
+  - `tekton-argocd-build-deploy-trigger-template.yaml` - contains the definition for the Tekton TriggerTemplate used this tutorial (in-depth explanations can be found inside).
+- `kustomization.yaml` - this is the main kustomization file (in-depth explanations can be found inside).
+
+**Important Note:**
+
+- Please bear in mind that the `configs` folder used by Kustomize contains **sensitive data**. So, make sure to not commit those files in your Git repository (you can use a `.gitignore` file for this purpose).
+
+Please follow below steps to create all required resources (Tekton CRDs, Knative CRDs, etc) in your Kubernetes cluster, via Kustomize:
+
+1. Clone the `container-blueprints` repo (if not already):
+
+    ```shell
+    git clone https://github.com/digitalocean/container-blueprints.git
+    ```
+
+2. Change directory to your local copy:
+
+    ```shell
+    cd container-blueprints
+    ```
+
+3. Edit and save each property file from the `DOKS-CI-CD/assets/manifests/tekton/configs` subfolder, making sure to replace the `<>` placeholders accordingly. For example you can use [VS Code](https://code.visualstudio.com):
+
+    ```shell
+    code DOKS-CI-CD/assets/manifests/tekton/configs/argocd/auth.env
+
+    code DOKS-CI-CD/assets/manifests/tekton/configs/docker/config.json
+
+    code DOKS-CI-CD/assets/manifests/tekton/configs/github/pat.env
+    ```
+
+    **Hints:**
+      - To obtain Argo CD admin password, you can use below command:
+
+        ```shell
+        kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+        ```
+
+      - To obtain your DigitalOcean docker registry `read/write` credentials, you can use below command and write results directly in the `config.json` file:
+
+        ```shell
+        doctl registry docker-config --read-write tekton-ci > DOKS-CI-CD/assets/manifests/tekton/configs/docker/config.json
```

Comment:
Maybe a placeholder should be added here for the registry name since it's parameterised now.

Reply:
Thanks. Fixed in latest commit.